org.rdkit.knime.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.rdkit.knime.feature"
       label="RDKit Nodes Feature"
-      version="4.3.0.qualifier"
+      version="4.3.1.qualifier"
```

Comment:
If you merge this as planned with the RDKit binaries changes, you may just increment the version number together with the binaries change as discussed there. All version numbers should be in sync at the end.

Reply:
yeah, these changes were made as part of the RDKit binaries merge